test/python/circuit/test_registerless_circuit.py

```diff
@@ -513,3 +515,133 @@ def test_bit_conditional_multiple_gates(self):
         expected.h(qreg[1]).c_if(creg[1], False)
         expected.cx(qreg[1], qreg[0]).c_if(creg1[0], True)
         self.assertEqual(circuit, expected)
+
+
+@ddt
+class TestQuantumCircuitFindBit(QiskitTestCase):
+    """Test cases for QuantumCircuit.find_bit."""
+
+    @data(Qubit, Clbit)
+    def test_bit_not_in_circuit(self, bit_type):
+        """Verify we raise if the bit has not been attached to the circuit."""
+        qc = QuantumCircuit()
+        bit = bit_type()
+
+        with self.assertRaisesRegex(CircuitError, r""):
+            qc.find_bit(bit)
+
+    @data(Qubit, Clbit)
+    def test_registerless_bit_constr(self, bit_type):
```

Comment:
```suggestion
    def test_registerless_bit_constructor(self, bit_type):
```
(I wasn't initially sure if it was "constructor" or "constraint".)